File: SPAComidas/src/app/models/usuario.ts
export class Usuario {

    public id:number;
    public user:string;
    public password:string;
    public rol:string;//Admin o Cliente

    constructor(id:number, user:string, password:string, rol:string) {
      this.id = id;
      this.user = user;
      this.password = password;
      this.rol = rol;
  }
}

File: databaseComidas/src/main/java/com/edu/co/usuario/Usuario.java
package com.edu.co.usuario;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Usuario {

    @Id
    @GeneratedValue
    private long id;

    @Column(unique = true, name = "usua")
    private String user;

    private String password;
    
    private String rol;

    public long getId() {
        return id;
    }
    
    public void setId(long id) {
        this.id = id;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
    
    public String getRol() {
        return rol;
    }

    public void setRol(String rol) {
        this.rol = rol;
    }
}
